src/sentry/api/endpoints/event_ai_suggested_fix.py

```diff
@@ -295,6 +306,7 @@ def get(self, request: Request, project, event_id) -> Response:
         # Check the OpenAI access policy
         policy = get_openai_policy(request.organization)
         policy_failure = None
+        stream = request.GET.get("stream") == "yes"
```

Comment:
should we move this code to be after line 325?

Reply:
is this also the change the front end needs to have in the request? don't we want it to always be true?